scss/components/_state-screen.scss

```diff
@@ -1,3 +1,21 @@
+.oo-state-screen-audio {
+  height: 100%;
+
+  .oo-state-screen-audio-title {
+    @include vendor-prefixes (transition, $fade-up-transition);
+    pointer-events: none;
+    width: 100%;
+    font-size: 1em;
+
+    .oo-state-screen-audio-info {
```

Comment:
Another personal opinion but I prefer to avoid more than one level of nesting. This style will result in  a rule like '.oo-state-screen-audio .oo-state-screen-audio-title .oo-state-screen-audio-info` which has high specificity and is hard to override. If you had to add a media query for this element, for example, you'd have to recreate this hierachy in order for the style to take effect

Reply:
Will take a look